lambdas/backend/express-server.js

```diff
@@ -235,14 +238,14 @@ app.put('/marketplace-subscriptions/:usagePlanId', (req, res) => {
     marketplace.resolveCustomer(params, (err, data) => {
         if (err) {
             console.log(`marketplace error: ${JSON.stringify(err)}`)
-            res.status(400).json(err.data.message)
+            res.status(400).json(err.message)
         } else {
             console.log(`marketplace data: ${JSON.stringify(data)}`)
 
             // persist the marketplaceCustomerId in DDB
             // this is used when the subscription listener receives the subscribe notification
             const marketplaceCustomerId = data.CustomerIdentifier
-            customersController.updateCustomerMarketplaceId(cognitoIdentityId, marketplaceCustomerId, error, success)
+            customersController.updateCustomerMarketplaceId(cognitoIdentityId, marketplaceCustomerId, error, subscribeCustomerToUsagePlan)
```

Comment:
Can you follow-up with a fix for https://github.com/awslabs/aws-api-gateway-developer-portal/issues/47

The listener event could fire before this is committed to DDB